gradio/templates/frontend/index.html

```diff
@@ -67,7 +67,7 @@
 		<script>window.gradio_config = {{ config | toorjson }};</script>
 
 		<script
-			src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.9/iframeResizer.contentWindow.min.js"
+			src="https://cdn.jsdelivr.net/npm/iframe-resizer@4.3.9/js/iframeResizer.min.js"
```

Comment:
No errors in the console with this change?

Reply:
no errors, the console (in firefox) shows a succesful get